cmd/cmdutils.go

```diff
@@ -103,3 +105,110 @@ func validateStoragePath(client *occlient.Client, storagePath, componentName, ap
 	}
 	return nil
 }
+
+// printStorageInComponent prints all the storage in a given component of the application
+func printStorageInComponent(client *occlient.Client, applicationName string, componentName string) {
+	//these flags are initially false, will be true if we find any mounted or unmounted storage respectively
+	hasMounted := false
+	hasUnmounted := false
+	//defining the column structure of the table
+	tabWriterMounted := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
+	tabWriterUnmounted := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
+	//create headers of mounted and unmounted storage table
+	fmt.Fprintln(tabWriterMounted, "NAME", "\t", "SIZE", "\t", "PATH")
+	fmt.Fprintln(tabWriterUnmounted, "NAME", "\t", "SIZE")
+
```

Comment:
All the above looks a bit too clustered together. Add a newline before each of the comments to space it out a bit.